roles/set_openstack_containers/templates/update_env_vars.sh.j2

```diff
@@ -36,7 +36,7 @@ oc get csv -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} \
    {% for key, value in containers_dict.items() -%}
    | jq '(.spec.install.spec.deployments[0].spec.template.spec.containers[1].env[] | select(.name=={{ '\"' + key + '\"' }})) |= (.value={{ '\"' + value + '\"' }})' \
    {% if operator_name == 'openstack' -%}
-   | jq '(.spec.install.spec.deployments[1].spec.template.spec.containers[1].env[] | select(.name=={{ '\"' + key + '\"' }})) |= (.value={{ '\"' + value + '\"' }})' \
+   | jq '(.spec.install.spec.deployments[1].spec.template.spec.containers[0].env[] | select(.name=={{ '\"' + key + '\"' }})) |= (.value={{ '\"' + value + '\"' }})' \
```

Comment:
Why hardcoding it when we can do something like this?
`.spec.install.spec.deployments[0].spec.template.spec.containers[] | select(.name != "kube-rbac-proxy")`